queries/ocaml/highlights.scm

```diff
@@ -124,7 +124,10 @@
 
 "%" @punctuation.special
 
-["(" ")" "[" "]" "{" "}" "[|" "|]" "[<" "[>"] @punctuation.bracket
+; try not to override highlighting for the unit constant
+(parenthesized_expression ["(" ")"] @punctuation.bracket)
```

Comment:
if that works it's ok. If not there is also the posibility to use `(#set! priority 101)`

Reply:
Thanks, I'll try It as soon as I get some time at the computer

Reply:
You're right, current approach leaves some parens unhilighted, trying your approach

Reply:
This works, thanks!